check-interval.rst

```diff
@@ -0,0 +1,8 @@
+CheckInterval Option
+====================
+
+Values: 1 - (int) size
```

Comment:
Is this in seconds?
Also (int) size is defined to be at least 32bit. Using a number here might be easier to understand to non-programmers. :-)
Maybe 2^32?